src/main/java/com/uber/cadence/serviceclient/auth/AdminJwtAuthorizationProvider.java

```diff
@@ -38,10 +37,13 @@ public AdminJwtAuthorizationProvider(RSAPublicKey publicKey, RSAPrivateKey priva
 
   @Override
   public byte[] getAuthToken() {
+    final Instant now = Instant.now();
     final JWTCreator.Builder jwtBuilder = JWT.create();
+    int JWT_TTL_SECONDS = 60 * 10;
     jwtBuilder.withClaim("admin", true);
-    jwtBuilder.withClaim("ttl", 60 * 10);
-    jwtBuilder.withIssuedAt(Date.from(Clock.systemUTC().instant()));
+    jwtBuilder.withClaim("ttl", JWT_TTL_SECONDS);
```

Comment:
can you update PR description with details around how you tested this?

Reply:
Expanded existing unit test to check if ExpiresAt is the same as IssuedAt + TTL